feature-libs/cart/_index.scss

```diff
@@ -20,3 +19,9 @@ $useLatestStyles: true;
 @import './quick-order/index';
 @import './wish-list/index';
 @import './import-export/index';
+
+@import 'cxbase/blocks/modal';
```

Comment:
let's include the package path and place it under the spartacus/styles/scss/core to bundle them

`@import '~@spartacus/styles/scss/cxbase/blocks/modal';`